tests/python/relay/test_op_level3.py

```diff
@@ -247,6 +247,7 @@ def verify_reshape(shape, newshape, oshape):
         assert zz.checked_type == relay.ty.TensorType(oshape, "float32")
 
         func = relay.Function([x], z)
+        check_grad(func)
```

Comment:
I think it might be a good idea to separate this test out since gradient is implemented separately from the op

Reply:
This allow us to get the same coverage for both test_op_level3 and test_op_grad_level3. I think we should move toward this style.

Reply:
True. I think grad implementations could also use some refactoring to move around; it would be more convenient implementation-wise to put grad implementations next to their forward op implementations as well.

Reply:
That also sounds good. Tensor_grad is getting big.